### ISSUE TITLE
Wrong key is not rejected when attaching signatures

### ISSUE THREAD

**Issue by concordium-cl** [Concordium/concordium-desktop-wallet#72]
**Bug Description**
A wrong key could be attached to a governance proposal without showing an error. The proposal then showed an error when it was submitted to chain.

**Expected Result**
Signature is rejected already when proposer tries to add it to a proposal.

**Actual Result**
Signature was attached.

**Versions**
- DW 1.1.6

**Comment by concordium-cl** [comment]
@orhoj: Could you pls assess bug, so we can talk about it Monday afternoon? Thx.

**Comment by orhoj** [comment]
We used to have this check, but it was removed as part of making it possible to do proposals offline. But I cannot remember why this check was removed as part of that, so there are two options:
1. We had a good reason for removing it, and we will remember when we add it again.
2. It was removed by accident.
Having the check only means that the proposer has to be connected to a node when they add a signature, and I guess that is okay?
